service/service.py

```diff
@@ -0,0 +1,50 @@
+"""
+Promotion Service
+
+Paths:
+------
+GET / - Returns a welcome message and 200 code
+"""
+import os
+import sys
+import logging
+from flask import Flask, jsonify, request, url_for, make_response, abort
+from flask_api import status  # HTTP Status Codes
+# from werkzeug.exceptions import NotFound
```

Comment:
why is this commented out? do we not need it anymore?

Reply:
I commented it out cause I didn't use it in my codes, but we might need it afterwards.